src/src/components/Widgets/DataExport.tsx

```diff
@@ -52,6 +68,12 @@ export function DataExport({
         >
           {commonText('exportToTsv')}
         </Button.White>
+        <Button.White
+          disabled={durations === undefined || calendars === undefined}
+          onClick={() => createExportToJson()}
+        >
+          {'Export To Json'}
```

Comment:
Can you please add this string to the dictionary: https://github.com/maxxxxxdlp/calendar-plus/blob/242b830b8857010d012bf6c1e1977a181803e0e2/src/src/localization/common.tsx#L80